dashboard/app/views/levels/editors/_evaluation_question.html.haml

```diff
@@ -0,0 +1,7 @@
+:scss
+  .script_text textarea { width: 100%; height: 100px}
```

Comment:
The space is wrong, but haml is actually expecting the height at width to be unquoted